app/webpacker/css/admin/order_cycles.scss

```diff
@@ -62,3 +62,25 @@ form.order_cycle {
     }
   }
 }
+
+#linked-schedule-warning-modal {
```

Comment:
Are any of these styles improvements that could be added to the shared styles?
https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/components/modal_component/modal_component.scss